### ISSUE TITLE
The value range of dropout parameters and recurrent_dropout parameters of GRU/LSTM/SimpleRNN functions

### ISSUE THREAD

**Issue by PhyllisJi** [tensorflow/tensorflow#61256]
### Issue type

Documentation Bug

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

source

### TensorFlow version

tf2.12.0

### Custom code

Yes

### OS platform and distribution

MacOs

### Python version

3.9

### Current behavior?

The document describes the dropout parameters of the GRU/LSTM/SimpleRNN function and the recurrent_dropout parameters are floating-point numbers with values ranging from 0 to 1, but the program can still run normally if the number is not between 0 and 1.
#### Document

| `dropout`           | Float between 0 and 1. Fraction of the units to drop for the linear transformation of the inputs. Default: 0. |
| ------------------- | ------------------------------------------------------------ |
| `recurrent_dropout` | Float between 0 and 1. Fraction of the units to drop for the linear transformation of the recurrent state. Default: 0. |

### Standalone code to reproduce the issue

```shell
from tensorflow import keras


def bilstm(num_units=25, input_shape=10):
# bilstm input layer
    input_tensor = keras.Input(shape=input_shape)
# bilstm hidden layer
    x = keras.layers.Embedding(input_dim=985, output_dim=10, input_length=8)(input_tensor)
    x = keras.layers.AlphaDropout(rate=0.1, noise_shape=None)(x)
    y = keras.layers.LSTM(units=num_units, return_sequences=False, recurrent_dropout=0.1)
    x = keras.layers.GRU(units=21, stateful=False, dropout=-0.6477691805726197, recurrent_dropout=-0.6477691805726197)(x)
# bilstm output layer
    output_tensor = x
    model = keras.models.Model(inputs=input_tensor, outputs=output_tensor)
    return model


if __name__ == "__main__":
    bilstm().summary()
```

### Relevant log output

```shell
Model: "model"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
=================================================================
 input_1 (InputLayer)        [(None, 10)]              0         
                                                                 
 embedding (Embedding)       (None, 10, 10)            9850      
                                                                 
 alpha_dropout (AlphaDropout  (None, 10, 10)           0         
 )                                                               
                                                                 
 gru (GRU)                   (None, 21)                2079      
                                                                 
=================================================================
Total params: 11,929
Trainable params: 11,929
Non-trainable params: 0
```
```

**Comment by sushreebarsa** [comment]
@PhyllisJi Thank you for raising this issue!
I was able to replicate the issue as provided [here](https://colab.research.google.com/gist/sushreebarsa/32c22320c0b7877f4fde23ce1270818d/61256.ipynb).
Could you please share the link of the documentation where you are proposing the modifications?

**Comment by BiophiliaSWDA** [comment]
Both the dropout parameter and the recurrent_dropout parameter take values between 0 and 1 in the document.
[GRU](https://www.tensorflow.org/api_docs/python/tf/keras/layers/GRU)
[LSTM](https://www.tensorflow.org/api_docs/python/tf/keras/layers/LSTM)
[SimpleRNN](https://www.tensorflow.org/api_docs/python/tf/keras/layers/SimpleRNN)
